keras/layers/recurrent.py

```diff
@@ -38,19 +61,21 @@ def __init__(self, input_dim, output_dim,
         if weights is not None:
             self.set_weights(weights)
 
-    def _step(self, x_t, h_tm1, u):
+    def _step(self, x_t, mask_tm1, h_tm1, u):
```

Comment:
No, it represents the mask of the previous timestep. We use that mask value to decide whether to include h_tm1 in our calculation of h_t. If mask_tm1 is 0 then h_tm1 is invalid.